values.yaml

```diff
@@ -231,33 +234,92 @@ zabbixagent:
 zabbixweb:
   # -- Enables use of **Zabbix Web**
   enabled: true
+  replicaCount: 1
+  # -- set permissive podAntiAffinity to spread replicas over cluster nodes if replicaCount>1
+  pod_anti_affinity: true
   resources: {}
   image:
     # -- Zabbix web Docker image name
-    repository: zabbix/zabbix-web-apache-pgsql
+    repository: zabbix/zabbix-web-nginx-pgsql
     # -- Tag of Docker image of Zabbix web
-    tag: ubuntu-6.0.4
+    #tag: ubuntu-6.0.5
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line can be removed, correct?

Reply:
commit 3f529c7